### PR TITLE
[FLOSS T2] Documentation: Update packaging instructions for Fedora

### PR DESCRIPTION
**Pull request by 0x6178656c**
I updated the documentation explaining how to create a package on Fedora.

## Basics

- [x] Short descriptions of your changes are in the release notes
      (added as entry in `doc/news/_preparation_next_release.md` which
      contains `_(my name)_`)
      **Please always add something to the release notes.**
- [x] Details of what you changed are in commit messages
      (first line should have `module: short statement` syntax)
- [ ] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildservers are happy. If not, fix **in this order**:
  - [ ] add a line in `doc/news/_preparation_next_release.md`
  - [ ] reformat the code with `scripts/dev/reformat-all`
  - [ ] make all unit tests pass
  - [ ] fix all memleaks
- [x] The PR is rebased with current master.

## Checklist

- [ ] I added unit tests for my code
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation (see [Documentation Guidelines](https://www.libelektra.org/devgettingstarted/documentation))
- [ ] I fixed all affected decisions (see [Decision Process](https://www.libelektra.org/decisions/decision-process))
- [ ] I added code comments, logging, and assertions as appropriate (see [Coding Guidelines](https://www.libelektra.org/devgettingstarted/coding))
- [ ] I updated all meta data (e.g. README.md of plugins and [METADATA.ini](https://master.libelektra.org/doc/METADATA.ini))
- [ ] I mentioned [every code](/.reuse/dep5) not directly written by me in [reuse syntax](https://reuse.software/)

## Review

- [ ] Documentation is introductory, concise, good to read and describes everything what the PR does
- [ ] Examples are well chosen and understandable
- [ ] Code is conforming to [our Coding Guidelines](https://master.libelektra.org/doc/CODING.md)
- [ ] APIs are conforming to [our Design Guidelines](https://master.libelektra.org/doc/DESIGN.md)
- [ ] Code is consistent to [our Design Decisions](https://master.libelektra.org/doc/decisions)

## Labels

- [ ] Add the "work in progress" label if you do not want the PR to be reviewed yet.
- [ ] Add the "ready to merge" label **if the basics are fulfilled** and no further pushes are planned by you.

### CODE REVIEW

File: doc/INSTALL.md [hunk 1]
```diff
@@ -253,14 +253,35 @@ sudo apt-get -f install
...
+Run following command in the `build/package` directory:
 
 ```sh
 sudo yum localinstall *
 ```
 
+Depending on the system configuration one might also need to configure and run `ldconfig`, for instance
+
+```sh
+sudo echo "usr/local/lib" >> /etc/ld.so.conf
```
Comment (joni1993) [inline doc/INSTALL.md:281]: Shouldn't it be  `/usr/local/lib`?

**Review by joni1993 — CHANGES_REQUESTED** [review]

File: doc/INSTALL.md (hunk 1 above)
Comment (0x6178656c) [reply doc/INSTALL.md:281]: Yes, thanks for noticing this!

**Comment by Dynamichost96** [comment]
## Documentation is introductory, concise, good to read and describes everything what the PR does

Commit messages are good, and Documentation is excellent and understandable.  There seem to be multiple CI-Tests that fail, one is due to invalid Links and the Second one looks like you modified a source file of a man page, but did not commit the updated man page. The fix for the Second error is to be found in Details

## Examples are well chosen and understandable
No new examples were provided

##  Code is conforming to [our Coding Guidelines](https://master.libelektra.org/doc/CODING.md)
New code is conforming to the Guideline

##  APIs are conforming to [our Design Guidelines](https://master.libelektra.org/doc/DESIGN.md)
No new API where added

##  Code is consistent to [our Design Decisions](https://master.libelektra.org/doc/decisions)
New code is consistent to the Design Decisions

Review has been done together with @Bujuhu

**Comment by flo91** [comment]
Great update to the documentation!
Please rebase with current master to get rid of the error about the updated man page source file.

**Pull request merged**